package.json

```diff
@@ -122,7 +122,6 @@
     "transformIgnorePatterns": [
       "[/\\\\]node_modules[/\\\\].+\\.(js|jsx|mjs)$"
     ],
-    "module": "src/index.js",
```

Comment:
This config option prints out a warning when running tests with `jest` so I've removed it